src/commcare_cloud/ansible/roles/elasticsearch/templates/check_snapshot_status.py.j2

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+import time
+import argparse
+from elasticsearch import Elasticsearch
+from elasticsearch.client import SnapshotClient
+from datetime import datetime, date
+from datetime import timedelta
+
+def get_snapshot_client():
+    es = Elasticsearch('{{ groups.elasticsearch.0 }}', timeout=3600)
+    snapshot_client = SnapshotClient(es)
+    return snapshot_client
+
+def get_snapshots(client):
+    snapshots = list()
+    today = date.today()
+
+    s_list = client.get('{{ es_repository_name }}','_all')
+    for snapshot_number in range(len(s_list['snapshots'])):
```

Comment:
you could just iterate over the snapshots directly: `for snapshot in s_list['snapshots']:`